index.html

```diff
@@ -3,9 +3,9 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title> CSD120 Matthew Burgan</title>
+    <title> CSD120 - Matthew Burgan</title>
 </head>
-<Hello,World!">
-
+<body>
+    Hello,World!
 </body>
```

Comment:
When adding content to an element, don't replace the opening tag name.  Instead, add the content between the opening and closing tags